meson_options.txt

```diff
@@ -0,0 +1,3 @@
+option('introspection', type: 'boolean',
```

Comment:
Why would one ever want to disable introspection for this library?

Reply:
I do not know, a C program that wants to link with its own copy of the library but does not want to depend on gobject introspection to build it